app.js

```diff
@@ -0,0 +1,75 @@
+const hideList = document.querySelector(`#hideList`);
+const description = document.querySelector('#description');
+const descriptionInput = document.querySelector('#descriptionInput');
+const descriptionButton = document.querySelector('#descriptionButton');
+const addItemInput = document.querySelector('#addItemInput');
+const addItemButton = document.querySelector('#addItemButton');
+const myList = document.querySelector(`#myList`);
+
+// kod z cwiczenia
+//zmienilem event.target.tagName == 'BUTTON' na event.target.className === 'removeButton'
+// dodalem class 'removeButton' do elementów button w HTML wewnatrz li
+
+const listElements = document.getElementsByTagName('li');
+const listDiv = document.querySelector(`.list`);
+
+
+
+
+listDiv.addEventListener('click', (event) =>{
+	if(event.target.className === 'removeButton') {
+		let li = event.target.parentNode;
+		let ul = li.parentNode;
+		ul.removeChild(li);
+	}
+});
+
+// koniec kodu z cwiczenia
+
+
+// zmiana opisu listy
+descriptionButton.addEventListener('click', () => {
```

Comment:
Skoro w linijce `19` nadajesz `eventListener` na `listDiv` to mógłbyś go wykorzystać również w przypadku `descriptionButton` robiąc kolejny warunek pod ten konkretny button i tam wrzucić logikę z dodawaniem opisu i czyszczeniem `input`

Reply:
Wtedy całą logikę można rozwiązać przy pomocy `switch` zamiast klasycznych `else / if`